### ISSUE TITLE
[Feature Request] Add Raw Input to Death and Binocular cameras

### ISSUE THREAD

**Issue by RecklessGlue540** [ThirteenAG/GTAIV.EFLC.FusionFix#456]
### What should happen

Since they're still affected by the framerate...

### Checklist

- [X] Check the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already implemented.
- [X] [Search for other requests](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same feature.

**Comment by Tomasak** [comment]
Binocular camera behave this way, because its meant to be unsmooth like when you use them IRL.

**Comment by RecklessGlue540** [comment]
> Binocular camera behave this way, because its meant to be unsmooth like when you use them IRL.

Yeah, but still, if you move the mouse slowly, the deadzone (if it is actually deadzone, feels more like they didn't even adapt the camera for mouse controls lol) is very bad...
Perhaps make it slower by nature regardless of how fast you move the mouse and without deadzone or console stick control remnants or whatever it is, it may be having...
